.github/workflows/grpc.yml

```diff
@@ -0,0 +1,67 @@
+name: grpc-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  grpc-testsuite-1_37_1:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: grpc
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      #- name: Install dependencies
+      #  run: pip install --upgrade pip setuptools wheel
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+      #- name: Set env
+      # run: echo ::set-env name=RELEASE_VERSION::${GITHUB_REF#refs/*/v}
+      - uses: actions/checkout@v2
+        with:
+          repository: grpc/grpc
+          ref: master
+         # submodules: true
+      #- name: Build
+      #  run: |
+      #    mkdir -p ${{ runner.tool_cache }}/grpc/1.37.0/x64
+      #    mkdir -p cmake/build
+      #    cd cmake/build
+      #    cmake -DgRPC_INSTALL=ON -DgRPC_SSL_PROVIDER=package \
+      #      -DgRPC_BUILD_TESTS=OFF -DBUILD_SHARED_LIBS=ON \
+      #      -DCMAKE_INSTALL_PREFIX=${{ runner.tool_cache }}/grpc/${{ 1.37.0 }}/x64 ../..
+      #    make -j 2
+      #    make install
+      #- name: Echo version
+      #  run: |
+      #    echo ${{ github.ref }}
+      #    ${{ runner.tool_cache }}/grpc/${{ 1.37.0 }}/x64/bin/protoc --version
+      #- name: Make tarball
+      #  run: |
+      #    tar -C ${{ runner.tool_cache }}/grpc/${{ 1.37.0 }}/x64 -zcf grpc-${{  1.37.0 }}-${{ matrix.os }}-x64.tar.gz .
+      #- name: Upload tarball to release
+      ##  uses: svenstaro/upload-release-action@v2
+       # with:
+       ##   repo_token: ${{ secrets.GITHUB_TOKEN }}
+        #  file: grpc-${{ 1.37.0 }}-${{ matrix.os }}-x64.tar.gz
+        #  tag: ${{ github.ref }}
+      #install grpc
+      #- name: Install grpc
+      #  run: pip install grpcio
+      #install grpc tools
+      #- name: Install grpc tools
+      #  run: python -m pip install grpcio-tools
+      - name: Inject ddtrace
+        run: pip install ../ddtrace
```

Comment:
given the above missing `path: grpc`, it would make sense that this is the part that is failing when running the GitHub action.